lib/child-compiler.js

```diff
@@ -74,6 +74,16 @@ class HtmlWebpackChildCompiler {
    * @returns {Promise<{[templatePath: string]: { content: string, hash: string, entry: WebpackChunk }}>}
    */
   compileTemplates (mainCompilation) {
+    const {
```

Comment:
Now the api follows `webpack@5` interface

and typing enabled :)

<img src="/rubeniskov/webpack-import-plugins/raw/master/docs/demo_typing.gif" style="max-width:100%;">

Reply:
Really cool 👍